AU2/plugins/custom_plugins/CompetencyPlugin.py

```diff
@@ -109,6 +120,26 @@ def set_default_competency_deadline_answer(self, htmlResponse):
         GENERIC_STATE_DATABASE.arb_int_state[self.plugin_state["DEFAULT"]] = new_val
         return [Label(f"[COMPETENCY] Updated game start to {new_game_start} and extension to {new_val}")]
 
+    def ask_auto_competency(self):
+        return [
+            InputWithDropDown(
+                identifier=self.html_ids["Auto Competency"],
+                title="Select Auto Competency Mode",
+                options=self.AUTO_COMPETENCY_OPTIONS,
+                selected=GENERIC_STATE_DATABASE.arb_state.get(self.plugin_state["Auto Competency"], "Manual")
+            )
+        ]
+
+    def answer_auto_competency(self, htmlResponse):
+        mode = htmlResponse[self.html_ids["Auto Competency"]]
+        GENERIC_STATE_DATABASE.arb_state[self.plugin_state["Auto Competency"]] = mode
+        response = [Label(f"[COMPETENCY] Auto Competency Mode set to {mode}")]
+        if mode != "Manual" and not GENERIC_STATE_DATABASE.plugin_map.get("AttemptPlugin", False):

```

Comment:
The config parameter for the `AttemptPlugin` is in the `CompetencyPlugin`. This makes me think we do not need the `AttemptPlugin` and that this functionality can be in the `CompetencyPlugin`. When the parameter is set to an appropriate mode, the `CompetencyPlugin` can avoid asking the `Attempts` question?